cellrank/tl/_lineages.py

```diff
@@ -27,28 +24,27 @@ def lineages(
     """
```

Comment:
@Marius1311 this docstring needs an update - I think it's still from the time when we've used metastable probabilities from GPCCA as lineaege probabilitiles.

Reply:
I like the way you changed the docstring. I just added a few lines and pushed directly to master.